sw/device/lib/testing/pwrmgr_testutils.c

```diff
@@ -0,0 +1,29 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/pwrmgr_testutils.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_pwrmgr.h"
+#include "sw/device/lib/testing/check.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+void pwrmgr_testutils_enable_low_power(dif_pwrmgr_t *pwrmgr, uint32_t wakeups) {
+  // Enable low power on the next WFI with default settings.
+  // All clocks and power domains are turned off during low power.
+  dif_pwrmgr_domain_config_t config;
```

Comment:
I wonder if this should be an input too. For some tests, we may be only interested in clock gating low power mode rather than full pd.